example/local/cog_example.py

```diff
@@ -0,0 +1,214 @@
+"""
+
+This example isn't recommended in production.
+Just use it only to learn the structure.
+This code is not perfect and can be complex.
+
+
+Additionally, remote is recommended for efficient performance management.
+Just know there is a way to do this.
+
+"""
+
+import logging
+
+import discord
+from discord.ext.commands import Bot
+
+import discodo
+
+
+
+class MusicBot(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.Audio = discodo.self.AudioManager()
+
+    @commands.Cog.listener()
+    async def on_socket_response(self, payload):
+        self.Audio.discordDispatch(payload)
+
+    @commands.Cog.listener()
+    async def on_ready(self, c):
```

Comment:
why there is c parameter?

```suggestion
    async def on_ready(self):
```